app/controllers/templates_controller.rb

```diff
@@ -274,6 +274,16 @@ def admin_update
 
       @template.description = params["template-desc"]
       @template.links = JSON.parse(params["template-links"]) if params["template-links"].present?
+
+      if current_user.org.funder?
+        if params[:template_visibility].nil?
+          @template.visibility = Template.visibilities[:publicly_visible]
+        else
+          @template.visibility = Template.visibilities[:organisationally_visible]
+        end
+      else
+        @template.visibility = Template.visibilities[:organisationally_visible]
```

Comment:
This else seems redundant to me. If I understood well, only funders are able to change the visibility of a template.

Reply:
It is preventing the value from being nil. Will refactor though to make the code less verbose